CosmosContracts/juno-tools
# refactor: implement strict eslint and prettier configs

## Description

This chonker PR refactors the eslint and prettier config which includes reformatting all source codes. Main reason is to ensure strictness, safety, and tidyness.

## Changes

List any technical changes.

- [x] [create eslint and prettier packages](https://github.com/CosmosContracts/juno-tools/commit/da43018394fd0e7043347bf7714a491c0ff7d01e)
- [x] [dedupe packages and integrate configs](https://github.com/CosmosContracts/juno-tools/commit/d53c91de8157edfbbcbe220ab75259b123a54de3)
- [x] [lint and manual fixes](https://github.com/CosmosContracts/juno-tools/commit/97dcde8178460848e06b92a018730cba49f2c4ba)

## Screenshots

\-

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- `yarn install`
- make sure running dev and build works as usual

## Links

Add links to Figma files, documentation, etc.

\-
